ambry-messageformat/src/main/java/com.github.ambry.messageformat/MessageFormatRecord.java

```diff
@@ -244,7 +245,7 @@ public static void serializeHeader(ByteBuffer outputBuffer, long totalSize, int
       outputBuffer.putLong(crc.getValue());
       Logger logger = LoggerFactory.getLogger("MessageHeader_Format_V1");
       logger.trace("serializing header : version {} size {} blobpropertiesrecordrelativeoffset {} " +
-          "deleterecordrelativeoffset {} " + "usermetadatarecordrelativeoffset {} blobrecordrelativeoffset {} crc {}",
+              "deleterecordrelativeoffset {} " + "usermetadatarecordrelativeoffset {} blobrecordrelativeoffset {} crc {}",
```

Comment:
hmmm... for some reason both formatters really like adding this indent. I guess it's because it's part of the first method argument's expression.